cardano-api/src/Cardano/Api/IPC/NtcVersionOf.hs

```diff
@@ -0,0 +1,21 @@
+module Cardano.Api.IPC.NtcVersionOf
+  ( NtcVersionOf (..)
+  , MinNodeToClientVersion
+
+  -- *** Error types
+  , UnsupportedNtcVersionError(..)
+  ) where
+
+import           Data.Eq (Eq)
+import           Text.Show (Show)
+import           Ouroboros.Network.NodeToClient.Version (NodeToClientVersion (..))
+
+-- | The query 'a' is a versioned query, which means it requires the Node to support a minimum
+-- Node-to-Client version.
+class NtcVersionOf a where
```

Comment:
I think we should expand the name to `NodeToClientVersionOf`

Reply:
Renamed the type class and moved the module to `Cardano.Api.IPC.Version`